content/docs/1.6.0/references/helm-values.md

```diff
@@ -5,257 +5,253 @@ weight: 5
-| persistence.backingImage.dataSourceType | string | `nil` | Specify the data source type for the backing image used in Longhorn StorageClass. If the backing image does not exists, Longhorn will use this field to create a backing image. Otherwise, Longhorn will use it to verify the selected backing image. |
-| persistence.backingImage.enable | bool | `false` | Set backing image for Longhorn StorageClass |
-| persistence.backingImage.expectedChecksum | string | `nil` | Specify the expected SHA512 checksum of the selected backing image in Longhorn StorageClass |
-| persistence.backingImage.name | string | `nil` | Specify a backing image that will be used by Longhorn volumes in Longhorn StorageClass. If not exists, the backing image data source type and backing image data source parameters should be specified so that Longhorn will create the backing image before using it |
-| persistence.defaultClass | bool | `true` | Set Longhorn StorageClass as default |
-| persistence.defaultClassReplicaCount | int | `3` | Set replica count for Longhorn StorageClass |
-| persistence.defaultDataLocality | string | `"disabled"` | Set data locality for Longhorn StorageClass. Options: `disabled`, `best-effort` |
-| persistence.defaultFsType | string | `"ext4"` | Set filesystem type for Longhorn StorageClass |
-| persistence.defaultMkfsParams | string | `""` | Set mkfs options for Longhorn StorageClass |
-| persistence.defaultNodeSelector.enable | bool | `false` | Enable Node selector for Longhorn StorageClass |
-| persistence.defaultNodeSelector.selector | string | `""` | This selector enables only certain nodes having these tags to be used for the volume. e.g. `"storage,fast"` |
-| persistence.migratable | bool | `false` | Set volume migratable for Longhorn StorageClass |
-| persistence.reclaimPolicy | string | `"Delete"` | Define reclaim policy. Options: `Retain`, `Delete` |
-| persistence.recurringJobSelector.enable | bool | `false` | Enable recurring job selector for Longhorn StorageClass |
-| persistence.recurringJobSelector.jobList | list | `[]` | Recurring job selector list for Longhorn StorageClass. Please be careful of quotes of input. e.g., `[{"name":"backup", "isGroup":true}]` |
-| persistence.removeSnapshotsDuringFilesystemTrim | string | `"ignored"` | Allow automatically removing snapshots during filesystem trim for Longhorn StorageClass. Options: `ignored`, `enabled`, `disabled` |
+| persistence.backingImage.dataSourceParameters | string | `nil` | Data source parameters of a backing image used in a Longhorn StorageClass. You can specify the JSON string equivalent of a Java map. (Example: `'{\"url\":\"https://backing-image-example.s3-region.amazonaws.com/test-backing-image\"}'`) |
```

Comment:
> "the JSON string equivalent of a Java map"

Suggest the original one: "a json string of a map"
 
I think  is more general in helm chart and no need to mention Java because most of the language have map and are all in the same structure. Users of this document should knows how to use helm and should be engineer.

"a json string of a map" means encoding the map struct into json string.